### ISSUE TITLE
[Question] File Mapping And Virtual Address Space support for plugins

### ISSUE THREAD

**Issue by Kingwolf123** [callofduty4x/CoD4x_Server#231]
Hi guys, server is crashing when loading a plugin which uses File Mapping and then crashes specifically when accessing the virtual Address Space. The code is fully tested on a test .so loaded into testapp (separate testing app for checking code).

Sample code of the plugin.so is implemented in the OnTenSeconds() event.
Opening a file handle to the a mapped file executes successfully, but traversing through the supposed virtual Address Space crashes the server.
Is this thing supported by cod4x?

**Comment by D4edalus** [comment]
it's quite hard to follow talking about imaginary code. if you are talking about the winapi functions for filemapping no, its not explicitely supported. but i wouldn't know why it needs explicit support by us either.
